python/ray/util/spark/cluster_init.py

```diff
@@ -127,7 +127,8 @@ def wait_until_ready(self):
                 # in this case, raise error directly.
                 if self.background_job_exception is not None:
                     raise RuntimeError(
-                        "Ray workers have exited."
+                        "Ray workers failed to start. You can check root "
+                        "cause from exception's `__cause__` attribute."
```

Comment:
What does this message assumes? Does this message assume the exception is caught and and a user has the exception instance?

Reply:
You can just say `Ray workers failed to start`.